python/cuml/tests/test_make_classification.py

```diff
@@ -117,6 +117,9 @@ def test_make_classification_informative_features():
             signs = np.sign(cp.asnumpy(X))
             signs = signs.view(dtype="|S{0}".format(signs.strides[0]))
             unique_signs, cluster_index = np.unique(signs, return_inverse=True)
+            # NumPy 2 has a behavior change (maybe regression) for the inverse shape
+            # https://github.com/numpy/numpy/issues/26738. This always works:
+            cluster_index = cluster_index.reshape(-1)
```

Comment:
```suggestion
            signs = signs.view(dtype="|S{0}".format(signs.strides[0])).ravel()
            unique_signs, cluster_index = np.unique(signs, return_inverse=True)
```
Saw this in scikit-learn, and it is a nicer pattern that doesn't feel like something that may need removal again.
(And this particular case I don't want to revert in NumPy, unlike the one affecting umap!)